scss/checkout/checkout.scss

```diff
@@ -1,6 +1,8 @@
 // variables
 $header-background-color: #444444;
 $footer-backgroundColor: #444444;
+$primary-color: #c7b198;
+$secondary-color: #929292;
```

Comment:
Someone will have to clean up the scss files too eventually, probably do a big code cleanup later.